prosody/rootfs/defaults/prosody.cfg.lua

```diff
@@ -1,3 +1,5 @@
+{{ $LOG_LEVEL := .Env.LOG_LEVEL | default "info" }}
```

Comment:
please document this and pass it in docker-compose

Reply:
done